iceoryx_examples/singleprocess/single_process.cpp

```diff
@@ -119,8 +119,8 @@ int main()
     iox::RouDiConfig_t defaultRouDiConfig = iox::RouDiConfig_t().setDefaults();
     iox::roudi::IceOryxRouDiComponents roudiComponents(defaultRouDiConfig);
 
-    iox::roudi::RouDi roudi(roudiComponents.m_rouDiMemoryManager,
-                            roudiComponents.m_portManager,
+    iox::roudi::RouDi roudi(roudiComponents.rouDiMemoryManager,
```

Comment:
Ok, I hope we can now universally adopt the rule of not using m_ prefix for public member variables in classes/structs.

Reply:
Only for [structs](https://github.com/eclipse-iceoryx/iceoryx/blob/master/CONTRIBUTING.md#naming-conventions) ;)